event-service-dto/pom.xml

```diff
@@ -17,4 +17,16 @@
         <maven.compiler.target>11</maven.compiler.target>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
why do you need the h2 dependency in the dto module?

Reply:
> why do you need the h2 dependency in the dto module?

done